app/javascript/components/patient/Patient.js

```diff
@@ -68,665 +91,622 @@ class Patient extends React.Component {
     return (
       <React.Fragment>
         <Row id="monitoree-details-header">
-          <Col className="mt-1">
-            <div className="h3">
+          <Col sm={12}>
+            <h3>
               <span className="pr-2">
                 {`${this.props.details.first_name ? this.props.details.first_name : ''}${
                   this.props.details.middle_name ? ' ' + this.props.details.middle_name : ''
                 }${this.props.details.last_name ? ' ' + this.props.details.last_name : ''}`}
               </span>
               {this.props?.dependents && this.props?.dependents?.length > 0 && <BadgeHOH patientId={String(this.props.details.id)} location={'right'} />}
-            </div>
+            </h3>
+            <div className="clearfix"></div>
           </Col>
-          <Col md="auto">
+          <Col sm={12}>
             <div className="jurisdiction-user-box">
-              <Row id="jurisdiction-path">
-                <Col>
-                  <b>Assigned Jurisdiction:</b> {this.props.jurisdiction_path || '--'}
-                </Col>
-              </Row>
-              <Row id="assigned-user">
-                <Col>
-                  <b>Assigned User:</b> {this.props.details.assigned_user || '--'}
-                </Col>
-              </Row>
+              <div id="jurisdiction-path">
+                <b>
+                  <span className="d-none d-md-inline">Assigned</span> Jurisdiction:
+                </b>{' '}
+                {this.props.jurisdiction_path || '--'}
+              </div>
+              <div id="assigned-user">
+                <b>Assigned User:</b> {this.props.details.assigned_user || '--'}
+              </div>
             </div>
           </Col>
         </Row>
-        <Row className="pt-md-4 mx-1 mb-2">
-          <Col id="identification" md="12">
+        <Row>
+          <Col id="identification" lg={14} className="col-xxl-12">
```

Comment:
Not a critique here, just a 👍 at this `col-xxl`. I feel like bootstrap doesnt have enough breakpoints.

Reply:
custom breakpoints for the win!